composer.json

```diff
@@ -26,7 +26,6 @@
         "doctrine/common": "^2.5",
         "doctrine/inflector": "~1.1.0",
         "drupal/coder": "^8.2.11",
-        "drush/drush": "^9.1.0",
```

Comment:
I would expect this to break a lot of things. Will drush no longer live `vendor/bin` for any project? I think BLT relies on it being there.